recipes/nomkl/meta.yaml

```
@@ -0,0 +1,24 @@
+{% set version = "1.0" %}
+
+package:
+  name: nomkl
+  version: {{ version }}
+
+
+build:
```

I think we want something like

```suggestion
build:
  track_features
    - nomkl
```

to make sure this package is not installed by accident into an env?

---

I don't think that's necessary. It will make it harder to depend on this pavkage